tools/test/topology/test-multi-playback.m4

```diff
@@ -110,5 +110,13 @@ DAI_CONFIG(TEST_DAI_TYPE, TEST_DAI_PORT, 0, TEST_DAI_LINK_NAME,
 		      SSP_CONFIG_DATA(TEST_DAI_TYPE, TEST_DAI_PORT,
 				      TEST_SSP_DATA_BITS, TEST_SSP_MCLK_ID)))
 
+DAI_CONFIG(TEST_DAI_TYPE, 2, 1, SSP2-Codec,
+	   SSP_CONFIG(TEST_SSP_MODE,
+		      SSP_CLOCK(mclk, TEST_SSP_MCLK, codec_mclk_in),
+		      SSP_CLOCK(bclk, TEST_SSP_BCLK, codec_slave),
+		      SSP_CLOCK(fsync, 48000, codec_slave),
+		      SSP_TDM(2, TEST_SSP_PHY_BITS, 3, 3),
+		      SSP_CONFIG_DATA(TEST_DAI_TYPE, 2,
+				      TEST_SSP_DATA_BITS, TEST_SSP_MCLK_ID)))
```

Comment:
From my experiment, this is crucial because we cannot use the same port and link name for two playback DAIs. So I put a hack here to configure SSP2 as well since from testbench SSP5 will be used by default (TEST_DAI_PORT=5)